src/js/utils/CreateEvent.js

```diff
@@ -0,0 +1,14 @@
+export function createEvent(event_type) {
```

Comment:
We could use the polyfill as specified here: https://developer.mozilla.org/en-US/docs/Web/API/CustomEvent/CustomEvent#Polyfill

This way the CreateEvent function could be removed entirely.